person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def __is_of_age?
+    return unless @age >= 18
+  end
+
+  def can_use_services?
+    if __is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+end
```

Comment:
- [ ] [Optional] It would be much better professionally if you make the `can_use_service?` method like that:
```ruby
def can_use_services?
    of_age? || @parent_permission
end
```